packages/manager/apps/telecom/src/app/telecom/telephony/line/line-main.html

```diff
@@ -89,13 +89,12 @@
             <oui-header-tabs-item
                 href="{{LineCtrl.faxLink}}"
                 active="LineCtrl.faxLink === LineCtrl.currentActiveLink()"
-                data-disabled="LineCtrl.loading.init || !(LineCtrl.line.isPlugNFax || LineCtrl.fax) || LineCtrl.line.isTrunk()"
+                data-disabled="LineCtrl.loading.init || !(LineCtrl.line.isPlugNFax || LineCtrl.fax || LineCtrl.line.hasFaxCapabilities) || LineCtrl.line.isTrunk()"
```

Comment:
i think better if we write all condition in JS function.
```suggestion
                data-disabled="!LineCtrl.isActiveFaxLineTab()"
```